rotkehlchen/tasks/calendar.py

```diff
@@ -0,0 +1,43 @@
+from typing import NamedTuple
+
+from rotkehlchen.api.websockets.typedefs import WSMessageType
+from rotkehlchen.db.calendar import CalendarEntry
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.user_messages import MessagesAggregator
+
+
+class CalendarNotification(NamedTuple):
+    """Basic reminder information along the calendar event linked to it"""
+    event: CalendarEntry
+    reminder_id: int | None
+    secs_before: int | None
+
+    def serialize(self) -> dict[str, str | int]:
+        data = self.event.serialize()
+        if self.secs_before != 0:  # it was a user created reminder instead of the basic one
```

Comment:
Why this distinction? It makes no sense. Even if it's created by the user if it's automatic it should always have a "secs_before" entry.